test/test_datasets.py

```diff
@@ -1757,5 +1657,92 @@ def inject_fake_data(self, tmpdir, config):
         return num_examples
 
 
+class Places365TestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.Places365
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(
+        split=("train-standard", "train-challenge", "val"),
+        small=(False, True),
+    )
+    _CATEGORIES = "categories_places365.txt"
+    # {split: file}
+    _FILE_LISTS = {
+        "train-standard": "places365_train_standard.txt",
+        "train-challenge": "places365_train_challenge.txt",
+        "val": "places365_val.txt",
+    }
+    # {(split, small): folder_name}
+    _IMAGES = {
+        ("train-standard", False): "data_large_standard",
+        ("train-challenge", False): "data_large_challenge",
+        ("val", False): "val_large",
+        ("train-standard", True): "data_256_standard",
+        ("train-challenge", True): "data_256_challenge",
+        ("val", True): "val_256",
+    }
+    # (class, idx)
+    _CATEGORIES_CONTENT = (
+        ("/a/airfield", 0),
+        ("/a/apartment_building/outdoor", 8),
+        ("/b/badlands", 30),
+    )
+    # (file, idx)
+    _FILE_LIST_CONTENT = (
+        ("Places365_val_00000001.png", 0),
+        *((f"{category}/Places365_train_00000001.png", idx)
+          for category, idx in _CATEGORIES_CONTENT),
+    )
+
+    @staticmethod
+    def _make_txt(root, name, seq):
+        file = os.path.join(root, name)
+        with open(file, "w") as fh:
+            for text, idx in seq:
+                fh.write(f"{text} {idx}\n")
+        return name, compute_md5(file)
```

Comment:
Computing the `md5` checksum was only needed in the old infrastructure, since we mocked the download and needed the checksum of the newly created files. In the new infrastructure we are mocking the complete download functionality, so there is not point in having these checksums.